src/blist.rs

```diff
@@ -243,30 +242,22 @@ impl<'a, T> Traverse<ring_buf::IterMut<'a, T>> for &'a mut RingBuf<T> {
     fn traverse(self) -> ring_buf::IterMut<'a, T> { self.iter_mut() }
 }
 
-/*
-impl<T> Traverse<ring_buf::MoveItems<T>> for RingBuf<T> {
-    fn traverse(self) -> ring_buf::MoveItems<T> { self.into_iter() }
+impl<T> Traverse<ring_buf::IntoIter<T>> for RingBuf<T> {
+    fn traverse(self) -> ring_buf::IntoIter<T> { self.into_iter() }
 }
-*/
 
 /// A by-ref iterator for a BList
-pub struct Items<'a, T: 'a> {
-    iter: AbsItems<dlist::Iter<'a, RingBuf<T>>, ring_buf::Iter<'a, T>>,
-}
-
+pub struct Iter<'a, T: 'a>
+    (AbsIter<dlist::Iter<'a, RingBuf<T>>, ring_buf::Iter<'a, T>>);
```

Comment:
I used to agree, but now that we have tuple-indexing it's slightly less tedious to use them.